### PR TITLE
Fix wrong permission name in biblio_ui_handler_delete::render()

### PR DESCRIPTION
**Pull request by shushu**

### CODE REVIEW

File: modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc [hunk 1]
```diff
@@ -8,7 +8,7 @@ class biblio_ui_handler_delete extends views_handler_field_entity {
   function render($values) {
     global $user;
 
-    if (user_access('delete biblio entries') || ($values->uid == $user->uid && user_access('delete own biblio entries'))) {
+    if (user_access('delete biblio content') || ($values->uid == $user->uid && user_access('delete own biblio content'))) {
```
Comment (amitaibu) [inline modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc:11]: delete biblio content => delete biblio content

Comment (shushu) [inline modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc:11]: I am not sure what you mean.

Comment (amitaibu) [inline modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc:11]: sorry, meant to write:

`delete biblio content` => `delete biblio`

Comment (shushu) [inline modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc:11]: hook_permissions() on biblio.module is all about "x biblio content". Do you want me to change them all ?

Comment (amitaibu) [inline modules/biblio_ui/includes/views/handlers/biblio_ui_handler_delete.inc:11]: I see,ok, leave as is for now.

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**